Reflections/ReflectionWeek6.md

```diff
@@ -0,0 +1,15 @@
+1. For this final week I worked super hard to finish everything that I could to be ready for the presentations.
+- I was able to finish getting the QR Code Sacanner to work and make a clickable link when it is clicked in the app and will take you to a new page.
+- I finished getting the color wheel to work and display the right color and now it will actually display the color of the text to be the same as the color of the wheel to truly reflect what it lands on even more.
+- I worked on making my app look better so I have taken out the color sliders and instead used photoshop to create a rainbow gradient background that is appealling.

```

Comment:
Would the rainbow background be everywhere in the pp?
I am asking because the wheel has several colors and if you have the colorful wheel in a rainbow gradient background, it might be too much color for the user. 

Just a thought, you can always see how it turns out and decide afterwards!